### ISSUE TITLE
Comments in Worklog / Quick History

### ISSUE THREAD

**Issue by Jogiio** [johannesjo/super-productivity#2266]
## Problem Statement

I'm always frustrated when I make my timerecording in the companys system / billing to the customer and I do not have the possibility in my Worklog / Quick History to see the comments (=Details) of the task in the Super Productivity App.

## :grey_question: Possible Solution

In each of the the two views (normaly I'm using the Worklog) please make it possible that I can see the earlier created comments to the task. 

## :arrow_heading_up: Describe alternatives you've considered

I considered none alternative solution.
## :heavy_plus_sign: Additional context

I knew none

**Comment by johannesjo** [comment]
I would also like to see a way to do this. A modal displaying all the task data, might be a good solution.

**Comment by Jogiio** [comment]
Not sure if I found a new functionality, which was done because of my request or if I overlooked this till now, but:
If I use the Worklog Export there is the possiblity to add the Column "Task Description" which is excatly what I needed.
<img width="935" alt="2022-11-14 08_48_39-Super Productivity" src="https://user-images.githubusercontent.com/84452087/201605181-150cf25b-fd33-4c48-97cc-08f881df4dba.png">
Not sure if Johannes knew about this and made another proposal or if his is now already fullfilled, but for me if fine like it is now. Thanks!
